packages/astro/astro-jsx.d.ts

```diff
@@ -1057,6 +1057,7 @@ declare namespace astroHTML.JSX {
 		poster?: string | undefined | null;
 		width?: number | string | undefined | null;
 		disablepictureinpicture?: boolean | string | undefined | null;
+		disableremoteplayback?: boolean | string | undefined | null;
```

Comment:
I think this should be on the media interface instead, as audio can also use this